discord/init.js

```diff
@@ -33,12 +37,50 @@ const fs = require('fs').readdirSync(normalizedPath).forEach( file => {
   console.log(`Loaded command from ${file}`)
 })
 
+let SlowmodeFilter = (msg) => {
+  if (guildCfg.slowmodedChannels.filter( smChannel => msg.channel.id === smChannel.id).length > 0) {
+    let channel = guildCfg.slowmodedChannels.filter( smChannel => msg.channel.id === smChannel.id)[0]
+    
+    let slowmodeTimePeriod = new Date( new Date().getTime() - channel.rate*1000 )
+    let slowmodeViolation = msg.channel.messages.cache.find(filteredMessages => {
+      return (filteredMessages.createdAt > slowmodeTimePeriod && filteredMessages.author === msg.author)
+    })
+
+    if (slowmodeViolation && slowmodeViolation !== msg) {
+      let timeRemaining = () => {
+        let diff = slowmodeViolation.createdAt.getTime() - slowmodeTimePeriod.getTime()
+        return `${diff/1000} seconds`
+      }
+      msg.author.send(`Sorry, but you sent a message in that channel recently and it's slowmoded! Here it is so you can try again in ${timeRemaining()}:\n${msg.content}`)
```

Comment:
do you know if this contains embeds? the current one with tillicev1 doesn't

Reply:
what do you mean

Reply:
if I send a message with a photo and it's slowmoded, do I get the photo back or only the text?